libraries/AC_Avoidance/AC_Avoid.cpp

```diff
@@ -27,7 +27,7 @@
  # define AP_AVOID_BEHAVE_DEFAULT AC_Avoid::BehaviourType::BEHAVIOR_SLIDE
 #endif
 
-#if APM_BUILD_TYPE(APM_BUILD_ArduCopter)
+#if APM_BUILD_TYPE(APM_BUILD_ArduCopter) || APM_BUILD_TYPE(APM_BUILD_Heli)
```

Comment:
I think having a combined macro for where we want either:
```
  #define APM_BUILD_COPTERHELI() (APM_BUILD_TYPE(APM_BUILD_ArduCopter) || APM_BUILD_TYPE(APM_BUILD_Heli))
```
I think that will make future bitrot less likely. This could go in AP_Vehicle_Type.h

Reply:
Done